examples/core/transform-feedback-instanced/app.js

```diff
@@ -151,34 +151,28 @@ const animationLoop = new AnimationLoop({
 
     const modelRender = new Model(gl, {
       id: 'Model-Render',
-      program: new Program(gl, {
-        vs: VS_DRAW,
-        fs: FS_DRAW
-      }),
+      vs: VS_DRAW,
+      fs: FS_DRAW,
       geometry: new Geometry({
         id: 'Model-Render-Geometry',
```

Comment:
Nit: Isn't the geometry automatically given an `id` by the model constructor? Or is that just for programs?

Always looking for ways to reduce the amount of lines / boilerplate in the examples.